app.py

```diff
@@ -0,0 +1,67 @@
+import subprocess
+import os
+from flask import Flask, jsonify, request, make_response
+from flask_sqlalchemy import SQLAlchemy
+from apscheduler.schedulers.background import BackgroundScheduler
+from secrets import SERVICE_ACCESS_TOKEN, MYSQL
+from datetime import datetime
+from models import Users, db
+
+scheduler = BackgroundScheduler()
+app = Flask(__name__)
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+app.config['SQLALCHEMY_DATABASE_URI'] = 'mysql+pymysql://{}:{}@{}/{}'.format(
```

Comment:
So it's using the same database as `groot-users-service` right? You should probably add a note of that somewhere, either in the README or in a comment as well.